.bazelci/presubmit.yml

```diff
@@ -36,9 +46,28 @@ tasks:
       - "--test_tag_filters=-skip-on-bazelci-ubuntu"
     test_targets:
       - "//..."
-  macos-smoke:
+  macos:
```

Comment:
We're adding macos testing on the root `//...`? Is that related to the headers?

Reply:
Not strictly related but good to add the coverage along with this change either way